dyn_em/module_diffusion_em.F

```diff
@@ -4278,6 +4278,10 @@ SUBROUTINE vertical_diffusion_2   ( ru_tendf, rv_tendf, rw_tendf, rt_tendf,   &
   CASE (0,2) ! with fixed surface heat flux given in the namelist
     heat_flux = config_flags%tke_heat_flux  ! constant heat flux value
                                             ! set in namelist.input
+    IF (config_flags%use_theta_m == 1)THEN
+      heat_flux = heat_flux * (1. + rvovrd * moist(i,kts,j,P_QV))
+    ENDIF
+
```

Comment:
This is outside the i and j loop - needs fixing for case (0,2)

Reply:
> This is outside the i and j loop - needs fixing for case (0,2)

you're totally right. I fixed this issue and also rearranged the code a bit to make it more readable.